hera_sim/tests/test_defaults.py

```diff
@@ -0,0 +1,42 @@
+from os.path import join
+import numpy as np
+from hera_sim.defaults import defaults
+from hera_sim.config import CONFIG_PATH
+from hera_sim.sigchain import gen_bandpass
+from hera_sim.noise import bm_poly_to_omega_p
+
+if defaults._version_is_compatible:
+    def test_config_swap():
+        defaults.set_defaults('h1c')
```

Comment:
you can probably make the method name just `set`, so the call will be `defaults.set('h1c')`. I think that reads more nicely.